src/test/javascript/spec/component/assessment-dashboard/assessment-dashboard-information.component.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { ArtemisTestModule } from '../../test.module';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import { MockTranslateService } from '../../helpers/mocks/service/mock-translate.service';
+import {
+    AssessmentDashboardInformationComponent,
+    AssessmentDashboardInformationEntry,
+} from 'app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component';
+import { DueDateStat } from 'app/course/dashboards/due-date-stat.model';
+import { PieChartModule } from '@swimlane/ngx-charts';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MockComponent, MockModule, MockPipe } from 'ng-mocks';
+import { TranslateService } from '@ngx-translate/core';
+import { SidePanelComponent } from 'app/shared/side-panel/side-panel.component';
+
+describe('AssessmentDashboardInformationComponent', () => {
+    let component: AssessmentDashboardInformationComponent;
+    let fixture: ComponentFixture<AssessmentDashboardInformationComponent>;
+    let setupGraphStub: jest.SpyInstance;
```

Comment:
This can be converted to a local variable in each test, because you're freshly settting it in every test and you don't access it in beforeEach or afterEach, if I'm not mistaken

Reply:
Done in bcd33ef